### ISSUE TITLE
counter_absolute does not work with the latest docker image

### ISSUE THREAD

**Issue by FSeidinger** [fhemberger/mqtt_exporter#36]
I'm using the docker image `fhemberger/mqtt_exporter:latest` which should be version 1.4.

When I use the an `counter_absolute` metric, the log file gives me the following warnings and does not publish the metric to prometheus.

```
WARNING: Metric type: counter_absolute, is not a valid metric type. Must be one of: ['gauge', 'counter', 'summary', 'histogram']
WARNING: Metric type: counter_absolute, is not a valid metric type. Must be one of: ['gauge', 'counter', 'summary', 'histogram']

```

As I saw in the tests, the `counter_absolute` should work. Do I do something wrong here?

My config looks like this:

```
# Config file for Mosquitto broker system metrics

# Metric definitions
metrics:
  - name:     'mqtt_broker'
    help:     'System events from broker'
    type:     'gauge'
    topic:    '$SYS/broker/#'
    label_configs:
      - source_labels:  ['__msg_topic__']
        target_label:   '__topic__'
      - source_labels:  ['__value__']
        regex:          '^(\d+([,.]\d*)?)$|^([,.]\d+)$'
        action:         'keep'

  - name:     'mqtt_broker_version'
    help:     'Mosquitto version (static)'
    type:     'gauge'
    topic:    '$SYS/broker/version'
    label_configs:
      - source_labels:  ['__msg_topic__']
        target_label:   '__topic__'
      - source_labels:  ['__value__']
        regex:          '^\D+((?:\d+[\.]?)+)$'
        target_label:   'version'
        replacement:    '\1'
        action:         'replace'
      - source_labels:  ['__value__']
        replacement:    '1'
        target_label:   '__value__'
        action:         'replace'

  - name:     'mqtt_broker_changeset'
    help:     'Mosquitto build changeset (static)'
    type:     'gauge'
    topic:    '$SYS/broker/changeset'
    label_configs:
      - source_labels:  ['__msg_topic__']
        target_label:   '__topic__'
      - source_labels:  ['__value__']
        target_label:   'changeset'
        action:         'replace'
      - source_labels:  ['__value__']
        replacement:    '1'
        target_label:   '__value__'
        action:         'replace'

  - name:     'mqtt_broker_timestamp'
    help:     'Mosquitto build timestamp (static)'
    type:     'gauge'
    topic:    '$SYS/broker/timestamp'
    label_configs:
      - source_labels:  ['__msg_topic__']
        target_label:   '__topic__'
      - source_labels:  ['__value__']
        target_label:   'timestamp'
        action:         'replace'
      - source_labels:  ['__value__']
        replacement:    '1'
        target_label:   '__value__'
        action:         'replace'

  - name:    'mqtt_broker_uptime_seconds'
    help:    'Mosquitto total uptime in seconds'
    type:    'counter_absolute'
    topic:   '$SYS/broker/uptime'
    label_configs:
      - source_labels:  ['__msg_topic__']
        target_label:   '__topic__'
      - source_labels:  ['__value__']
        regex:          '^(\d+) seconds$'
        replacement:    '\1'
        target_label:   '__value__'
        action:         'replace'
```

**Comment by fhemberger** [comment]
Could you try to use the image tag `v1.4.0` explicitly? We had an issue with the `latest` tag which still resolves to `v1.2.3`(see #30).

**Comment by FSeidinger** [comment]
Thanks,

that does the job.
